BaseTools/Conf/tools_def.template

```diff
@@ -896,7 +896,7 @@ NOOPT_*_*_OBJCOPY_ADDDEBUGFLAG     = --add-gnu-debuglink="$(DEBUG_DIR)/$(MODULE_
 
 DEFINE GCC_ALL_CC_FLAGS            = -g -Os -fshort-wchar -fno-builtin -fno-strict-aliasing -Wall -Werror -Wno-array-bounds -include AutoGen.h -fno-common
 DEFINE GCC_ARM_CC_FLAGS            = DEF(GCC_ALL_CC_FLAGS) -mlittle-endian -mabi=aapcs -fno-short-enums -funsigned-char -ffunction-sections -fdata-sections -fomit-frame-pointer -Wno-address -mthumb -fno-pic -fno-pie
-DEFINE GCC_LOONGARCH64_CC_FLAGS    = DEF(GCC_ALL_CC_FLAGS) -mabi=lp64d -fno-asynchronous-unwind-tables -fno-plt -Wno-address -fno-short-enums -fsigned-char -ffunction-sections -fdata-sections
+DEFINE GCC_LOONGARCH64_CC_FLAGS    = DEF(GCC_ALL_CC_FLAGS) -mabi=lp64d -fno-asynchronous-unwind-tables -Wno-address -fno-short-enums -fsigned-char -ffunction-sections -fdata-sections

```

Comment:
How is this supposed to work with GenFw? GOT entries are not covered by static relocations, so GenFw will not be able to generate PE/COFF relocations for them. Or does GenFw handle this already?

Reply:
Oh, I guess I didn't make it clear in my commit log:
1. The static relocations(`GOT_PC` and `PCALA`) have been handled in GenFw.
2. The option `fno-plt` only uses global function symbols, if enabled, the linker will force to create GOT when calling external functions, but edk2 will never call external functions in each EFI (it is statically linked), and if `fplt` is used, I think never create the PLT in edk2 either, because all symbols in each EFI seems to be local(statically linked), so `fno-plt` is not necessary, I remove it in this patch. BTW, other ARCHs don't turn on `fno-plt` either.

Can I update the commit log in next time?